streamer/usr/share/pulp/wsgi/streamer.tac

```diff
@@ -16,40 +16,10 @@ from pulp.plugins.loader import api as plugin_api
 
 DEFAULT_LOG_LEVEL = logging.INFO
 LOG_FORMAT_STRING = 'pulp_streamer: %(name)s:%(levelname)s: %(message)s'
-LOG_PATH = os.path.join('/', 'dev', 'log')
-
-
-def start_logging(config):
-    """
-    Configure the Pulp streamer syslog handler for the configured log level.
-    """
-    # Get and set up the root logger with our configured log level
-    try:
-        log_level = config.get('streamer', 'log_level')
-        log_level = getattr(logging, log_level.upper())
-    except (ConfigParser.NoOptionError, AttributeError):
-        # If the user didn't provide a log level, or if they provided an
-        # invalid one, let's use the default log level
-        log_level = DEFAULT_LOG_LEVEL
-    root_logger = logging.getLogger()
-    root_logger.setLevel(log_level)
-
-    # Set up our handler and add it to the root logger
-    if not os.path.exists(LOG_PATH):
-        print >> sys.stderr, "Unable to access to log, {log_path}.".format(log_path=LOG_PATH)
-        sys.exit(os.EX_UNAVAILABLE)
-
-    handler = CompliantSysLogHandler(address=LOG_PATH,
-                                     facility=CompliantSysLogHandler.LOG_DAEMON)
-    formatter = logging.Formatter(LOG_FORMAT_STRING)
-    handler.setFormatter(formatter)
-    root_logger.handlers = []
-    root_logger.addHandler(handler)
-
 
 # Initialization of application dependencies.
 streamer_config = load_configuration(DEFAULT_CONFIG_FILES)
-start_logging(streamer_config)
+start_logging()
```

Comment:
`start_logging` doesn't use this modules `LOG_FORMAT_STRING` which makes the logs of `streamer` indistinguishable by prefix from `server` logs.